_posts/2015/2016-07-20-fullstackcon-2016-retrospective.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "ePages goes FullStack JavaScript conference"
+date: "2016-07-20 12:00:00"
```

Comment:
In case Ryan will do the review today, we can change the date to today and I'll publish today :-)
Just let me know. Otherwise, tomorrow is also fine. Awesome that you made it that fast!